rpcs3/Emu/Cell/Modules/cellNetCtl.cpp

```diff
@@ -221,16 +221,15 @@ error_code cellNetCtlGetInfo(s32 code, vm::ptr<CellNetCtlInfo> info)
 	switch (code)
 	{
 	case CELL_NET_CTL_INFO_DEVICE: info->device = CELL_NET_CTL_DEVICE_WIRED; break;
-	// case CELL_NET_CTL_INFO_ETHER_ADDR: std::memset(info->ether_addr.data, 0xFF, sizeof(info->ether_addr.data)); break;
```

Comment:
why removed?

Reply:
It's handled above as it doesn't require status to be online.